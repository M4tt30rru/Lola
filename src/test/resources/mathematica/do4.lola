##Find do[##Any(_e),{##Literal(_times)}];
 ##run {
if 'x' not in locals():
 x=0
else:
 x=x+1
_doName='_i'+str(x)
 }
 ##replace for(int ##(_doName)=0;##(_doName)<##(_times);++##(_doName)) {##(_e);}

do[f(),{5}];
do[g(),{15}];
