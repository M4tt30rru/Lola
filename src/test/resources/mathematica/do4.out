for(int _i0=0;_i0<5;++_i0) {f();}
for(int _i1=0;_i1<15;++_i1) {g();}